src/util/bsearch.h

```diff
@@ -35,8 +35,10 @@ typedef int (*rsbcompare)(const void *s, const void *elem);
  * @param cmp the comparison function
  * @return `end`
  */
-static inline size_t rsb_ge(const void *arr, size_t narr, size_t elemsz, size_t begin, size_t end,
-                            const void *s, rsbcompare cmp) {
+static inline int rsb_gr(const void *arr, size_t narr, size_t elemsz, const void *s,
```

Comment:
why the name change?

Reply:
Now it check for Greater only and not GE